ynput/ayon-core
# Traypublisher: Actually enable Override Colorspace collector when it should be enabled

## Changelog Description

Fix CollectColorspace plug-in enabled state for tray publisher. Without this change it never shows since `enabled` is always False.

## Additional info

With this fix it shows again and seems to work:
![image](https://github.com/ynput/ayon-core/assets/2439881/fb4a0195-fa70-47bf-8e51-228727b77ef7)


## Testing notes:

1. Enable color management
2. Publish image or plate via traypublisher
3. The instance should have an "override colorspace" option
4. The overridden colorspace should work to set the interpreted published colorspace of the file.
